### DISCUSSION TITLE
ACOUSTIC AUTHENTICATION SYSTEM - ONLINE SECURITY

### DISCUSSION THREAD

**Discussion by ayzic45** [ggerganov/ggwave#58] [Ideas]
Hi, Georgi

To begin, I'd like to remark that your work is incredible; there are very little

 open-source data over sound libraries available.
I've adopted your technique into my project, and here are the connections to the transmitter and receiver sides of my web-based acoustic authentication system. When I'm finished, I'll export everything and upload it to GitHub so that anyone interested in seeing the code may do so easily.

**The aim was to replace the modern MFA authentication methods with acoustic authentication**, Instead of using an OTP from an SMS/Email, the code is collected from the database and played from the transmitter, and the receiver collects it and verifies if the two codes are the same depending on the currently signed-in user. I also created a script that tracks failed login attempts and suspends the account if a particular number of failed attempts are made until the user reaches the recovery phase, at which point the user should contact the administrator.

**Integration includes the use of PHP, JavaScript, HTML, CSS.**

**The links below have been updated as the new domain was used**
- Transmitter side: https://ukznacousticauth.site/transmitter.php
- Receiver side: https://ukznacousticauth.site/login.php
-Video: https://www.youtube.com/watch?v=Th2S27lk85k&ab_channel=ChannelK

Below are the images and the video demo link is attached.

![Screenshot from 2021-11-10 23-18-43](https://user-images.githubusercontent.com/58569721/144076909-e970ff55-9881-4e67-91eb-a2836abbb687.png)
![Screenshot from 2021-11-10 23-20-17](https://user-images.githubusercontent.com/58569721/144076917-4a22c46e-6f6b-4631-9c26-45691dffbf72.png)
![Screenshot from 2021-11-12 02-58-08](https://user-images.githubusercontent.com/58569721/144076921-13c0957a-98a0-45b7-b3b1-7e9791a9f6ae.png)
![3 attempts locked](https://user-images.githu
![recovery](https://user-images.githubusercontent.com/58569721/144077111-3dd00c4e-5105-4709-b41c-63b7fdb9609f.PNG)
busercontent.com/58569721/144077053-5724bb89-e922-43cd-a310-f7bbae13d3d8.PNG)
![Capture2](https://user-images.githubusercontent.com/58569721/144077073-6c608fae-3a4b-41a0-9c32-761257eeb775.PNG)

Video Demo: https://www.youtube.com/watch?v=Th2S27lk85k&ab_channel=ChannelK

**Comment by ggerganov** [comment]
Hi,

Interesting project! Would love to learn more about how it works.
What do you think are the benefits of such authentication approach compared to existing MFA? Do I still need an app in order to authenticate with this method? A video demonstration would be useful.

Keep us posted with the progress on this!

**Reply by ayzic45** [reply]
Security of online systems is being compromised by attackers, as the technology advances they are also gaining new tricks to gain unauthorized access, sometimes they do sim swaps on behalf of owners.

I will provide the report explaining how the system works(including the benefits of this method over the current ones) as soon I'm done with it, also the video demonstration will be available no later than this month.

at this point, no app required it a website for easy use,  maybe later I will develop the transmitter app.

**Comment by ayzic45** [comment]
Here is the video demo for the acoustic authentication system: https://youtu.be/Th2S27lk85k
